src/lazy/text/buffer.rs

```diff
@@ -3060,11 +2384,9 @@ mod tests {
             "_123",        // Leading underscore
             "0x0x5",       // Multiple 0x prefixes
             "0xx5",        // Multiple Xs after 0
-        ],
-        expect_incomplete: [
             "0x",          // Base 16 prefix w/no number
             "0b",          // Base 2 prefix w/no number
-        ]
+        ],
```

Comment:
🪧 Because these unit tests are all reading from fixed slices, the parser will never return `Incomplete`. Those inputs have been moved to the `expect_mismatch` sections. We have [a separate test suite](https://github.com/amazon-ion/ion-rust/blob/89d3206c050666877494e0bb7364b041f504c0f5/tests/detect_incomplete_text.rs#L67-L69) just for incompleteness detection anyway.